README.md

```diff
@@ -1,14 +1,114 @@
 # Experimental Synthetics Agent
 
 Synthetic Monitoring with Real Browsers.
-**Please note this is an unsupported experimental project. Expect things to break and change!** 
+**Please note this is an unsupported experimental project. Expect things to break and change!**
 
 Docs are forthcoming, feel free to browse the repository, but we ask that you refrain from asking questions about use till docs
 are published.
```

Comment:
```suggestion
Documentation is available on [elastic.co](https://www.elastic.co/guide/en/observability/current/synthetic-monitoring.html).

Have questions? Want to leave feedback? Visit the [Synthetics discussion forum](https://discuss.elastic.co/tags/c/observability/uptime/75/synthetics).
```

Reply:
Docs link is ready. Feel free to add it in however you'd like

Reply:
Sure, Thanks Brandon 🎉